tests/chainer_tests/functions_tests/math_tests/test_floor.py

```diff
@@ -60,6 +75,7 @@ class TestFloor(UnaryFunctionsTestBase):
     def make_data(self):
         x = numpy.random.uniform(-10.0, 10.0, self.shape).astype(self.dtype)
         gy = numpy.random.uniform(-1, 1, self.shape).astype(self.dtype)
+        self.ggy = numpy.random.uniform(-1, 1, self.shape).astype(self.dtype)
```

Comment:
Would you let `make_data` to return a triple of `(x, gy, ggy)` instead of assigning `ggy` to the instance variable here?

Reply:
Thanks, fixed.